README.md

```diff
@@ -1 +1,15 @@
 # hw01
+
+## Self Introduction
+My name is Yuwen Xiong. I am a second-year master student at **UChicago Harris School of Public Policy**. Below is the photo of mine:
+
+![Photo of Yuwen](LinkedIn-profile-smaller.jpg)
+
+If you want to know more about me, you can take a look at [my LinkedIn profile](https://www.linkedin.com/in/yuwenxiong/) .
```

Comment:
Great job making a link to your LinkedIn profile --- I was able to click on the link and it took me to your profile.